src/cryptoadvance/specter/devices/specter.py

```diff
@@ -173,6 +174,5 @@ def get_wallet_fingerprint(wallet):
     Unique fingerprint of the wallet -
     first 4 bytes of hash160 of its descriptor
     """
-    h256 = hashlib.sha256(get_wallet_qr_descriptor(wallet).encode()).digest()
-    h160 = hashlib.new("ripemd160", h256).digest()
+    h160 = hash160(get_wallet_qr_descriptor(wallet).encode()).digest()
```

Comment:
hash160 from embit doesn't need digest()

```suggestion
    h160 = hash160(get_wallet_qr_descriptor(wallet).encode())
```

Reply:
Thanks. I missed that.